src/template.rs

```diff
@@ -53,6 +56,40 @@ pub struct Template {
     #[serde(skip)]
     is_static: bool,
 
+    // <<<<<<< HEAD
+    // impl Template {
+    //     /// Returns `true` if this template string has a length of zero, and `false` otherwise.
+    //     pub fn is_empty(&self) -> bool {
+    //         self.src.is_empty()
+    //     }
+    //
+    //     pub fn new(src: &str) -> Result<Self, TemplateParseError> {
+    //         let (has_error, is_dynamic) = StrftimeItems::new(src)
+    //             .fold((false, false), |(error, dynamic), item| {
+    //                 (error || is_error(&item), dynamic || is_dynamic(&item))
+    //             });
+    //
+    //         let fields = get_fields(src);
+    //
+    //         for field in &fields {
+    //             if parse_target_path(field).is_err() {
+    //                 return Err(TemplateParseError::InvalidPathSyntax {
+    //                     path: field.to_owned(),
+    //                 });
+    //             }
+    //         }
+    //
+    //         if has_error {
+    //             Err(TemplateParseError::StrftimeError)
+    //         } else {
+    //             Ok(Template {
+    //                 has_fields: !fields.is_empty(),
+    //                 src: src.to_owned(),
+    //                 has_ts: is_dynamic,
+    //             })
+    //         }
+    //     }
+    // =======
```

Comment:
Did you mean to leave in these merge conflicts?

Reply:
:man_facepalming:  fixed